packages/flutter/lib/src/cupertino/interface_level.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../widgets/framework.dart';
+
+/// Indicates the visual level for a piece of content.
```

Comment:
We'll need to do a better job of explaining this; probably provide a link back to iOS13 UI guidelines too.